spec/features/remove_member_spec.rb

```diff
@@ -0,0 +1,130 @@
+require 'spec_helper'
+
+describe 'delete transaction', js: true do
+  let(:user)       { create :user }
+  let(:other_user) { create :user }
+  let!(:org)       { create :organization }
+
+  context 'when member with owner role' do
+    let!(:member_owner) { create :member, :owner, user: user, organization: org }
+
+    before do
+      sign_in member_owner.user
+      visit members_path
+    end
```

Comment:
@yunixon why not move this `before` block up, to the main `describe` block? Looks like you use it in all cases in this spec.